### PR TITLE
Added F722RE Variant to boards.txt, added F722RE ldscript.ld

### PR DESCRIPTION
**Pull request by geosmall**

### CODE REVIEW

**Comment by geosmall** [comment]
Add F722RE variant

**Comment by geosmall** [comment]
Try again to add F722RE variant

File: boards.txt [hunk 1]
```diff
@@ -2796,6 +2796,14 @@ GenF7.build.flags.fp=-mfpu=fpv4-sp-d16 -mfloat-abi=hard
 GenF7.build.series=STM32F7xx
 GenF7.build.cmsis_lib_gcc=arm_cortexM7lfsp_math
 
+# Generic F722RETx
```
Comment (fpistm) [inline boards.txt:2799]: I would add also the 3 other mcu's here:
https://github.com/stm32duino/Arduino_Core_STM32/blob/main/variants/STM32F7xx/F722R(C-E)T_F730R8T_F732RET/boards_entry.txt

**Review by fpistm — CHANGES_REQUESTED** [review]
Hi  @geosmall 
Thanks for the PR.
Could you add the 3 other mcu boards entries int the boards.txt?
It misses also the update of the README.md.

I guess it should be fine to add a default system core clock config to ensure USB clock is correctly set to 48 MHz in https://github.com/stm32duino/Arduino_Core_STM32/blob/949ecc80d0399b4a5ead733ff57be5f728a47c4d/variants/STM32F7xx/F722R(C-E)T_F730R8T_F732RET/generic_clock.c#L25

**Comment by fpistm** [comment]
README.md will be updated later else it will add merge conflict:
```patch
+| :yellow_heart: | STM32F722RC<br>STM32F722RE | Generic Board | **2.3.0** |  |
+| :yellow_heart: | STM32F730R8<br>STM32F732RE | Generic Board | **2.3.0** |  |
```

**Pull request merged**